plugins/00_dokku-standard/commands

```diff
@@ -34,6 +34,9 @@ case "$1" in
         ;;
 
       dockerfile)
+        # extract first port from Dockerfile
+        DOCKERFILE_PORT=$(grep EXPOSE Dockerfile | head -1 | awk '{ print $2 }')
+        [[ -n "$DOCKERFILE_PORT" ]] && dokku config:set $APP DOKKU_DOCKERFILE_PORT=$DOCKERFILE_PORT
```

Comment:
Good catch. I didn't see any issue because we guard against it on the first deploy.

https://github.com/progrium/dokku/blob/53f9974f6f905de5ec99163cc691f2493e6048e3/plugins/ps/commands#L86-L87

Let me rework the config portion.